commit a5452902a60c1513509be403c99ba6ba494f3e19 (from 773229b02049297adb4c67df303453892083215f)
Merge: 773229b e0830bf
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Fri Apr 15 19:56:24 2016 -0700

    Merge pull request #3516 from thinkyhead/rc_lcd_alert_only_on_error
    
    Don't use LCD alert unless error or alert level is reset later

diff --git a/Marlin/Marlin_main.cpp b/Marlin/Marlin_main.cpp
index d51f2f7..c0acebf 100644
--- a/Marlin/Marlin_main.cpp
+++ b/Marlin/Marlin_main.cpp
@@ -5851,7 +5851,7 @@ inline void gcode_M428() {
 
   if (!err) {
     sync_plan_position();
-    LCD_ALERTMESSAGEPGM(MSG_HOME_OFFSETS_APPLIED);
+    LCD_MESSAGEPGM(MSG_HOME_OFFSETS_APPLIED);
     #if HAS_BUZZER
       buzz(200, 659);
       buzz(200, 698);
diff --git a/Marlin/ultralcd.cpp b/Marlin/ultralcd.cpp
index b68a4e5..5d5845c 100644
--- a/Marlin/ultralcd.cpp
+++ b/Marlin/ultralcd.cpp
@@ -953,7 +953,7 @@ void lcd_cooldown() {
         mbl.set_zigzag_z(_lcd_level_bed_position++, current_position[Z_AXIS]);
         if (_lcd_level_bed_position == (MESH_NUM_X_POINTS) * (MESH_NUM_Y_POINTS)) {
           lcd_return_to_status();
-          LCD_ALERTMESSAGEPGM(MSG_LEVEL_BED_DONE);
+          LCD_MESSAGEPGM(MSG_LEVEL_BED_DONE);
           #if HAS_BUZZER
             buzz(200, 659);
             buzz(200, 698);

